etna/records/fields.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, iaid: str):
         self.__dict__["iaid"] = iaid
```

Comment:
If a `Record` can represent any type of record now (ie. with or without an `iaid` value), I don't think the custom field or the chooser should be using/storing the `iaid` value specifically, as that will no longer work in all circumstances. This is another reason why I suggest using a `id` throughout... we're not saying there cannot be other identifiers for different types of record - simply that there is a canonical property that does work in any circumstance.

Reply:
Same as mentioned previously: The current api response and attribute have the same name i.e. `iaid` until now, no other source is formalised.

Reply:
I'm really really confused :(

Reply:
Records without `iaid`, this needs further understanding and discussions. I believe we have tickets for them. Also I've mentioned about `primaryIdentifier` [here](https://github.com/nationalarchives/ds-wagtail/pull/645#discussion_r1115864254) . The circumstance of `iaid` has not changed until now. Therefore until then the custom field or chooser field should remain `iaid`. Hope this clarifies.